packages/developer-console-spa/src/components/configureSSI.module.css

```
@@ -0,0 +1,17 @@
+.theme-image {
+  transition: 120ms ease-in-out;
+  border: 2px solid #ececec;
+  border-radius: 4px;
+  width: 100%;
+  box-sizing: border-box;
+  cursor: pointer;
+}
+
+.text-bold {
+  font-weight: 700;
+}
+
+.theme-image[active-theme],
+.theme-image:hover {
+  border: 2px solid #0066cc;
```

Increase the border radius. The active block needs to be more prominent.
```suggestion
  border: 3px solid #0066cc;
```

---

Done